docs/guides/1.23.0-upgrade-guide.md

```diff
@@ -0,0 +1,25 @@
+---
+page_title: "Upgrade Guide 1.23.0"
+---
+
+# MongoDB Atlas Provider 1.23.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.23.0 has a number of new and exciting features.
+
+## New Resources, Data Sources, and Features
+
+- You can now leverage auto-scaling to scale the instance size of each individual shard independently for sharded and geo-sharded clusters defined with `mongodbatlas_advanced_cluster`. To learn more, see the [Advanced Cluster New Sharding Configurations - Use Auto-Scaling Per Shard](advanced-cluster-new-sharding-schema#use-auto-scaling-per-shard).
```

Comment:
nit: should we better use a full url so it always goes to the latest version (once 1.23.0 is not the latest version) as in the links below?